python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -239,13 +240,18 @@ def groups(*group_strs, include_sources: bool = False) -> "GroupsAssetSelection"
             include_sources (bool): If True, then include external assets matching the group in the
                 selection.
         """
-        check.tuple_param(group_strs, "group_strs", of_type=str)
-        return GroupsAssetSelection(selected_groups=group_strs, include_sources=include_sources)
+        check.tuple_param(group_strs, "group_strs", of_type=Optional[str])
```

Comment:
I didn't think you could use `Optional[]` in these check methods (you'd need to do of_type=(str, None)).